chainerx_cc/chainerx/routines/manipulation.h

```diff
@@ -49,4 +49,6 @@ std::vector<Array> Split(const Array& ary, int64_t sections, int8_t axis = 0);
 // Returns a set of arrays resulting from splitting the given array at the indices along the specified axis.
 std::vector<Array> Split(const Array& ary, std::vector<int64_t> indices, int8_t axis = 0);
 
+Array Swapaxes(const Array& ary, int8_t axis1, int8_t axis2);
```

Comment:
One last occurrence of `ary`.
```suggestion
Array Swapaxes(const Array& a, int8_t axis1, int8_t axis2);
```

Reply:
Hahah thanks! Nothing slips your eyes.

Reply:
Quick fix as always, thanks a lot :)